spec/rubocop/target_ruby_spec.rb

```diff
@@ -201,6 +201,21 @@
         end
       end
 
+      context 'when file reads the required_ruby_version from another file in an array' do
+        it 'uses the default target ruby version' do
+          content = <<~HEREDOC
+            Gem::Specification.new do |s|
+              s.name = 'test'
+              s.required_ruby_version = [">= \#{File.read('.ruby-version').rstrip}"]
+              s.licenses = ['MIT']
+            end
+          HEREDOC
```

Comment:
```suggestion
          content = <<~'HEREDOC'
            Gem::Specification.new do |s|
              s.name = 'test'
              s.required_ruby_version = [">= #{File.read('.ruby-version').rstrip}"]
              s.licenses = ['MIT']
            end
          HEREDOC
```

Reply:
@koic : Thank you for the review and the good suggestion! I have applied your suggestion both to the spec that I am adding in this PR (as you suggested) and also to the similar spec just above, which I added in #12732. (Unfortunately, this makes the diff in GitHub look a little bit more complicated than necessary.)